build.sh

```diff
@@ -1,6 +1,8 @@
 set -e
 cd site
 jekyll build -d ../content_tmp
+# Remove dangling references to the DejaVu fonts in the Javadoc stylesheets
+sed -i "" "/^@import/d; s/'DejaVu Sans',//g; s/'DejaVu Sans Mono',//g; s/'DejaVu Serif',//g" ../content_tmp/javadoc/*/stylesheet.css
```

Comment:
Removing all imports vs only those associated with dejavu.css seems a little heavy handed?
Maybe also only have a single pattern to match any current, or future, DejaVu font spec?  Sed seems to lack a non-greedy specifier but the following seems to work s/'DejaVu[^']*',*//g  -- match everything starting with 'DejaVu and all chars up the first single quote, and that single quote, and an optional comma (i.e., if it came at the end of the font spec)